### PR TITLE
chore(benchmarks): call-requests and call-results benchmarks

### PR DESCRIPTION
**Pull request by monoid**

### CODE REVIEW

**Review by raftedproc — COMMENTED** [review]
Do you need to update the global Cargo.lock in the context of this patch?

**Comment by monoid** [comment]
@raftedproc it is updated because someone forgot to update it in the previous commits.

File: benches/PERFORMANCE.json [hunk 1]
```diff
@@ -3792,255 +3874,667 @@
...
-          "58.938 MiB",
-          "58.938 MiB"
+          "59.000 MiB",
+          "59.000 MiB"
         ],
         "stats": {
           "air::runner::execute_air": {
             "common_prefix": "air",
-            "duration": "12.11ms",
+            "duration": "17.72ms",
```
Comment (raftedproc) [inline benches/PERFORMANCE.json:3912]: I am curious if this is a degradation and something we expect?

Comment (raftedproc) [reply benches/PERFORMANCE.json:3912]: Should be signatures processing overhead?

Comment (monoid) [reply benches/PERFORMANCE.json:3912]: CID verification.

**Pull request merged**